app/uk/gov/hmrc/helptosavefrontend/controllers/email/AccountHolderUpdateEmailAddressController.scala

```diff
@@ -140,15 +141,15 @@ class AccountHolderUpdateEmailAddressController @Inject() (val helpToSaveService
         (enrolmentStatus, maybeEmail) match {
           case (EnrolmentStatus.NotEnrolled, _) ⇒
             // user is not enrolled in this case
-            auditor.sendEvent(SuspiciousActivity(nino, "missing_enrolment"), nino)
+            auditor.sendEvent(SuspiciousActivity(Some(nino), "missing_enrolment"), nino)
             logger.warn("User was not enrolled", nino)
             internalServerError()
 
           case (EnrolmentStatus.Enrolled(_), None) ⇒
             // this should never happen since we cannot have created an account
             // without a successful write to our email store
             logger.warn("User was enrolled but had no stored email", nino)
-            auditor.sendEvent(SuspiciousActivity(nino, "missing_email_record"), nino)
+            auditor.sendEvent(SuspiciousActivity(Some(nino), "missing_email_record"), nino)
```

Comment:
Pls change logger.warn string to "SuspiciousActivity: user is enrolled but the HtS email record does not exist"